app/services/load_testing_urls_writer.rb

```diff
@@ -0,0 +1,43 @@
+class LoadTestingUrlsWriter
```

Comment:
Would prefer this not under `app` since it is for test purposes.  `lib` or other preferred